<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadiationDamageModelWidgetLayout</class>
 <widget class="QWidget" name="RadiationDamageModelWidgetLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RadiationDamageModelWidget</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="main_groupbox">
     <property name="title">
      <string>Radiation damage model</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="beta_over_gray_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>βÅ&lt;sup&gt;2&lt;/sup&gt;/MGy:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="beta_over_gray_ledit">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QLabel" name="gamma_over_gray_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>γ 1/MGy:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="gamma_over_gray_ledit">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2">
       <widget class="QLineEdit" name="sensetivity_ledit">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="sensetivity_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Sensetivity:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
